licenses/LICENCE-postgresql.txt

```diff
@@ -0,0 +1,24 @@
+PostgreSQL Database Management System
```

Comment:
Yes, though nit: "LICENSE" vs "LICENCE". We're using the US spelling consistently